index.html

```diff
@@ -63,11 +63,37 @@ <h2 id="modalTitle">Add New Item</h2>
               <span class="error" id="nameError"></span>
             </label>
 
+            <div class="formInputGroup">
             <label for="quantityInput">
               Quantity
               <input type="number" id="quantityInput" value="1" />
             </label>
 
+            <label for="quantityUnitInput">
+              Unit 
+              <select name="quantityUnitInput" id="quantityUnitInput">
+                <option value="unit">unit</option>
```

Comment:
Defaulting to unit seemed to make sense, but defaulting to no value is probably the best.